### ISSUE TITLE
Water-terminating glaciers

### ISSUE THREAD

**Issue by oberrauch** [OGGM/oggm-vas#11]
Frontal ablation of water terminating glaciers is not considered in the scaling model, and this should be properly handled (warnings, errors, ...?!)

**Comment by fmaussion** [comment]
No, we should do like Ben (and also OGGM sometimes) does: simply simulate them as "non tidewater" glaciers
